bluebrain/deployment/environments/applications_science.yaml

```diff
@@ -14,12 +14,14 @@ spack:
           - py-bluecellulab
           - py-bluepyefe
           - py-bluepyemodel
+          - py-bluepyemodelnexus
           - py-bluepymm
           - py-bluepyopt
           - py-bluepyparallel
           - py-connectome-manipulator
           - py-data-integrity-check
           - py-efel
+          - py-icselector
```

Comment:
do we need a separate module for this?

Reply:
We probably don't need one, unless @vandorp wants one?

Reply:
I removed it